rust/gui-client/README.md

```diff
@@ -75,6 +75,23 @@ If that doesn't work:
 - Click `New client secret`
 - Note down the secret value. This should be entered into the GitHub repository's secrets as `AZURE_CLIENT_SECRET`.
 
+## Installing on CentOS 9
+
+These instructions will move to the knowledge base once the first release supporting CentOS 9 is cut.
+
+1. Download RPM
+2. `sudo dnf install systemd-resolved` (Installing it explicitly prevents it from being auto-removed if Firezone is removed)
+3. `sudo dnf install epel-release` (Needed to get GNOME extensions)
+4. `sudo dnf install ./firezone-client-gui-*.rpm`
+5. `sudo usermod -aG firezone-client $USER`
```

Comment:
The sys-users stuff from systemd doesn't work on rpm based distros?

Reply:
It works, but this step is also needed on Ubuntu. It's the same as adding yourself to the `docker` group, I don't think installing a package is supposed to add users to groups

Reply:
Ah yes, got it mixed up!